### PR TITLE
Rename Profile package to install

### PR DESCRIPTION
**Pull request by aclevername**
### Description

This package and its contents are all related to installing the profile into your local repository. Naming makes slightly more sense and is in-line with the `ProfileInstallation` CRD

### Checklist
- [ ] Added tests that cover your change
- [ ] Added/modified documentation as required (such as the `README.md`)
- [ ] Made sure the title of the PR is a good description that can go into the release notes
- [ ] Added a `kind` label to the PR (e.g. `kind/feature`)

### CODE REVIEW

File: cmd/pctl/install.go [hunk 1]
```diff
@@ -179,7 +179,7 @@ func install(c *cli.Context) error {
 		gitRepoNamespace = split[0]
 		gitRepoName = split[1]
 	}
-	artifactsMaker := profile.NewProfilesArtifactsMaker(profile.MakerConfig{
+	artifactsMaker := install.NewInstaller(install.InstallerConfig{
```
Comment (Skarlso) [inline cmd/pctl/install.go:182]: `artifactsMaker` doesn't really fit any longer eh? :)

Comment (aclevername) [reply cmd/pctl/install.go:182]: dam thought I caught all of those! good spot :wink:

File: pkg/install/installer.go [hunk 2]
```diff
@@ -16,39 +16,41 @@ import (
...
 	GitRepoNamespace string
 	GitRepoName      string
 	ProfileName      string
 }
 
-// ProfilesArtifactsMaker creates a list of artifacts from profiles data.
-type ProfilesArtifactsMaker struct {
-	MakerConfig
-
+// Installer
```
Comment (Skarlso) [inline pkg/install/installer.go:41]: Don't forget to expand on this :)

File: pkg/install/installer.go [hunk 3]
```diff
@@ -16,39 +16,41 @@ import (
...
 	nestedName   string
 	profileRepos []string
 	cloneCache   map[string]string
 }
 
-// NewProfilesArtifactsMaker creates a new profiles artifacts maker.
-func NewProfilesArtifactsMaker(cfg MakerConfig) *ProfilesArtifactsMaker {
-	return &ProfilesArtifactsMaker{
-		MakerConfig: cfg,
+// NewInstaller creates a new profiles artifacts maker.
```
Comment (Skarlso) [inline pkg/install/installer.go:50]: profiles installer. :)

File: pkg/install/installer.go [hunk 4]
```diff
@@ -60,21 +62,22 @@ func NewProfilesArtifactsMaker(cfg MakerConfig) *ProfilesArtifactsMaker {
 	}
 }
 
-// Make generates artifacts without owners for manual applying to a personal cluster.
-func (pa *ProfilesArtifactsMaker) Make(installation profilesv1.ProfileInstallation) error {
-	artifacts, err := profilesArtifactsMaker(pa, installation)
+// Install installs the artifacts
```
Comment (Skarlso) [inline pkg/install/installer.go:65]: you gonna this one, aye? :D

File: pkg/install/installer_test.go [hunk 5]
```diff
@@ -182,15 +182,15 @@ var _ = Describe("Profile", func() {
 		Expect(os.RemoveAll(rootDir)).To(Succeed())
 	})
 
-	Context("Make", func() {
+	Context("Install", func() {
 		It("generates the artifacts", func() {
-			maker := profile.NewProfilesArtifactsMaker(profile.MakerConfig{
+			installer := install.NewInstaller(install.InstallerConfig{
```
Comment (Skarlso) [inline pkg/install/installer_test.go:187]: this feels frigging weird. waaay to many `install`s. :D Does config need to be `InstallerConfig`? Do we have another config in that package? I don't think we do.

**Review by Skarlso — APPROVED** [review]
LGTM, some typos and some changes on the comments. Well done, I agree with this refactor. :)

File: pkg/install/installer_test.go (hunk 5 above)
Comment (aclevername) [reply pkg/install/installer_test.go:187]: yeah I think jsut calling it `Config` makes sense, then tis `install.config`

**Pull request merged**